kodilla-testing/src/main/java/com/kodilla/testing/shape/ShapeCollector.java

```diff
@@ -0,0 +1,42 @@
+package com.kodilla.testing.shape;
+
+import java.util.ArrayList;
+
+public class ShapeCollector {
+ArrayList<Shape> shapes = new ArrayList<>();
+
+    public void addFigure(Shape shape) {
+       Shape shape = new Shape() {
+           @Override
+           public String getShapeName() {
+               return null;
+           }
+
+           @Override
+           public int getField() {
+               return 0;
+           }
+       }
```

Comment:
po co tworzyc Shape w metodzie addFigure ?
To usuniecia. Zostaw tylko shapes.add(shape)